database/database_test.go

```diff
@@ -62,46 +62,46 @@ var putAndAllTests = []struct {
 	{
 		"one domain",
 		[]DomainState{
-			{Name: "gmail.com", Status: StatusPending},
+			{Name: "gmail.test", Status: StatusPending},
```

Comment:
nit: it would be better for the renames in lines otherwise untouched by this PR to be in a separate PR that just does cleanup work. (You don't have to do the cleanup of existing code.)